### ISSUE TITLE
In-event filtering: Activity stream within an area shows foreign activities

### ISSUE THREAD

**Issue by lordyavin** [theCrag/website#2773]
The activitiy stream for the [Dachstein](https://www.thecrag.com/climbing/germany/schwaebische-alb/donautal/area/249412230) shows activities from other cliffs that are siblings, not children.

This should not be the case. Only activities on the node itself or it's children should be displayed.

![wrong_fead](https://cloud.githubusercontent.com/assets/1528540/26574684/918696c4-4522-11e7-8bc7-ea7494da11d6.jpg)

**Comment by lordyavin** [comment]
Looks like the rate if new issues gets higher and higher ;-) No response since 11 days.

**Comment by scd** [comment]
At the moment the rules are to bring in events that involve the area. Events are tied to the top level crag.

We made the decision, once an event was brought in not to then filter it to only the activity involving the area.

While doing extra filtering in an event may be an improvement, I am not 100% sure in all cases. 

To get the in-event display filtering working is a fair bit of work so we probably will not tackle this for a long time.

BTW, yes you are right, we are finding it difficult to find the time to get through everything at the moment. It will be the case until we can sort out some structural resource issues. 

Usually we will try and comment on an issue straight away, but if we don't it means that we got distracted and it has probably fallen off our radar. No harm in politely reminding us again :)
